lib/stdlib.ngs

```diff
@@ -2105,6 +2105,22 @@ section "Range" {
 	TEST 1 =~ 0..5
 	TEST 0 =~ 0..5
 	TEST 5 !~ 0..5
+
+	doc Checks that x is in the range.
+	doc %EX - ensure(3, 3..10)  # 3
+	F ensure(n:Int, r:NumRange){
+		n in r returns n
+    	result = cond {
+        		n < r.start and r.include_start r.start
+        		n < r.start and not(r.include_start) r.start + r.step
+        		n < r.end r.start
```

Comment:
Not sure about this one. Doesn't sound right. Don't have enough focus to investigate this now. Please think about this.

Reply:
Can I use comments to explain? I found this the esiest way to short the code